bin/dev

```diff
@@ -18,7 +18,7 @@ findBackendContainerId(){
 # args: commands to run in container
 runDockerExec(){
     findBackendContainerId container_id
-    cmd="docker exec -it -u $(id -u):$(id -g) $container_id "$@
+    cmd="docker exec -it -u $(id -u):$(id -g) -w /home/worker/app/server $container_id "$@
```

Comment:
Bin/dev commands are kindof broken - working on a fix in another pr, temporary solution here.

Reply:
Did this work? Nick tried bunch of things, so I wasn't sure what was the fix. I think the final fixture was running `pre-commit` locally outside of docker to set everything up.

Reply:
interesting. This is necessary for other commands to work. Didn't consider nick's problem when doing this.